src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -65,6 +68,63 @@ static char *IsolationLevelName[] = {
 };
 
 
+PG_FUNCTION_INFO_V1(start_management_transaction);
+PG_FUNCTION_INFO_V1(execute_command_on_other_nodes);
+PG_FUNCTION_INFO_V1(add_to_pg_dist_node_on_all_nodes);
+
+bool isManagementCommand = false;
+FullTransactionId outerXid;
+
+Datum
+start_management_transaction(PG_FUNCTION_ARGS)
```

Comment:
maybe `citus_internal_start_management_transaction`, since this is super-duper internal please-dont-call-yourself.